sdk/src/main/java/com/fingerprint/api/FingerprintApi.java

```diff
@@ -380,18 +380,96 @@ public ApiResponse<VisitorsGetResponse> getVisitsWithHttpInfo(String visitorId,
                                localVarHeaderParams, localVarCookieParams, localVarFormParams, localVarAccept, localVarContentType,
                                localVarAuthNames, localVarReturnType, false);
   }
+  public static class SearchEventsFilter {
+    
+    private String paginationKey;
+    private String visitorId;
+    private String bot;
+    private String ipAddress;
+    private String linkedId;
+    private Long start;
+    private Long end;
+    private Boolean reverse;
+    private Boolean suspect;
+
+    public String getPaginationKey() {
+      return paginationKey;
+    }
+
+    public SearchEventsFilter setPaginationKey(String paginationKey) {
+      this.paginationKey = paginationKey;
+      return this;
+    }
+    public String getVisitorId() {
+      return visitorId;
+    }
+
+    public SearchEventsFilter setVisitorId(String visitorId) {
+      this.visitorId = visitorId;
+      return this;
+    }
+    public String getBot() {
+      return bot;
+    }
+
```

Comment:
There is a line break between `getBot()` and `setBot()` but no line break between `setBot()` and `getIpAddress()` etc. It probably should be the opposite. Oversight or codegen limitation?

Reply:
Fair! Fixed.